commcare/src/main/resources/webapp/js/controllers.js

```diff
@@ -247,6 +249,7 @@
             copy.forwardSchema = config.forwardSchema;
             copy.forwardStubs = config.forwardStubs;
             copy.forwardCases = config.forwardCases;
+            $scope.oldName = copy.name;
```

Comment:
don't like the idea of storing this in scope, while everything else is returned by the method. Can't we return an object of {copy, oldName} instead?